docs/directives.md

```diff
@@ -0,0 +1,55 @@
+# @glint Directives
+
+Glint has several directives that can be used in template comments to control
+Glint's behavior. Additional comment text can follow directives to document
+their purpose.
+
+## @glint-ignore
+
+The `@glint-ignore` directive tells Glint to ignore the line that follows it.
+Glint will not report any errors encountered on the next line.
+
+Example:
+
+```hbs
+<MyComponent @expectedArg="foo" />
+
+{{! @glint-ignore: need to update signature to accept this arg }}
+<MyComponent @unexpectedArg="bar" />
```

Comment:
This maybe seems like a more compelling example for `expect-error` to me (though to be fair, that's how I feel about almost every situation where you'd use one of those directives 😅)

Realistically the only case where I use `@{ts,glint}-ignore` is where I'm dealing with compatibility stuff, e.g.

```hbs
{{! @glint-ignore: this doesn't typecheck in TS 4.6 due to a bug, but we still test against that version in CI }}
<MyComponent @arg={{this.arg}} />
```